caffeine@patapon.info/locale/uk.po

```diff
@@ -0,0 +1,417 @@
+msgid "Enable or disable the toggle in the quick settings menu"
+msgstr "Ввімкнути або вимкнути перемикач у панелі швидких налаштувань"
+
+#: preferences/displayPage.js:74
+msgid "Notifications"
+msgstr "Повідомлення"
+
+#: preferences/displayPage.js:75
+msgid "Enable notifications when Caffeine is enabled or disabled"
+msgstr "Ввімкнути повідомлення, коли \"Кофеїн\" увімкнено або вимкнено"
+
+#: preferences/displayPage.js:82
+msgid "Status indicator position"
+msgstr "Позиція піктограму стану"
+
+#: preferences/displayPage.js:83
+msgid "The position relative of indicator icon to other items"
+msgstr "Позиція відносна до піктограм інших об'єктів"
+
+#: preferences/generalPage.js:35
+msgid "General"
+msgstr "Загальні"
+
+#: preferences/generalPage.js:45
+msgid "Behavior"
+msgstr "Поведінка"
+
+#: preferences/generalPage.js:50
+msgid "Enable for fullscreen apps"
+msgstr "Увімкнути для повноекранних додатків"
+
+#: preferences/generalPage.js:51
+msgid "Automatically enable when an app enters fullscreen mode"
+msgstr "Автоматично вмикати коли додаток входить в режим повного екрану"
+
+#: preferences/generalPage.js:57
+msgid "Enable when an app is playing media"
+msgstr "Увімкнути коли додаток відтворює медіа"
+
+#: preferences/generalPage.js:58
+msgid "Automatically enable when an app reports playback"
+msgstr "Автоматично вмикати коли додаток відтворює медіа"
+
+#: preferences/generalPage.js:64
+msgid "Remember state"
+msgstr "Запам'ятовувати стан"
+
+#: preferences/generalPage.js:65
+msgid "Remember the last state across sessions and reboots"
+msgstr "Запам'ятовувати стан між сесіями та перезавантаженнями системи"
+
+#: preferences/generalPage.js:73 preferences/generalPage.js:85
+msgid "For apps on list"
+msgstr "Для додатків у списку"
+
+#: preferences/generalPage.js:75
+msgid "Pause and resume Night Light"
+msgstr "Поставити на паузу та продовжити \"Нічне світло\""
+
+#: preferences/generalPage.js:76
+msgid "Toggles the night light together with Caffeine's state"
+msgstr "Перемикати \"Нічне світло\" разом із станом \"Кофеїну\""
+
+#: preferences/generalPage.js:87
+#: schemas/org.gnome.shell.extensions.caffeine.gschema.xml:92
+msgid "Allow screen blank"
+msgstr "Дозволити притлумлення екрану"
+
+#: preferences/generalPage.js:88
+msgid "Allow turning off screen when Caffeine is enabled"
+msgstr "Дозволити вимикати екран коли ввімкнено \"Кофеїн\""
+
+#: preferences/generalPage.js:111
+msgid "Shortcut"
+msgstr "Клавіатурні скорочення"
+
+#: preferences/generalPage.js:119
+#: schemas/org.gnome.shell.extensions.caffeine.gschema.xml:102
+msgid "Toggle shortcut"
+msgstr "Скорочення для перемикання стану"
+
+#: preferences/generalPage.js:120
+msgid "Use Backspace to clear"
+msgstr "Використовуйте Backspace щоб очистити"
+
+#: preferences/generalPage.js:200 preferences/generalPage.js:232
+msgid "New accelerator…"
+msgstr "Нове скорочення…"
+
+#: preferences/timerPage.js:41
+msgid "Timer"
+msgstr "Таймер"
+
+# Не знаю як перекласти "preset".
```

Comment:
I don't know any Ukrainian I just used Google translate to sanity check the translations, did you need more context for preset? We can add a translators comment in the source code that gives more context for translators.